src/utils/retry_utils/inc/aduc/retry_utils.h

```diff
@@ -55,6 +55,9 @@ EXTERN_C_BEGIN
 #define DEFAULT_ENR_REQ_OP_MAX_JITTER_PERCENT (60)
 #define SETTING_KEY_ENR_REQ_OP_MAX_JITTER_PERCENT "maxJitterPercent"
 
+#define SETTING_KEY_POLLING_INTERVAL_SECONDS "pollingIntervalInSeconds"
+#define ADUC_DEFAULT_POLLING_INTERVAL_SECONDS (5 * 60)
```

Comment:
This might not be the right PR but aren't these apart of the configuration file (like the one in the example?). If so these access functions and the defaults should be inside of config_utils not the retry_utils